src/Controller/EnMarche/SummaryManagerController.php

```diff
@@ -37,14 +39,39 @@ public function indexAction()
         $this->disableInProduction();
 
         $member = $this->getUser();
+        $summary = $this->get(SummaryManager::class)->getForAdherent($this->getUser());
+        $pathImage = 'images/'.$summary->getMemberUuid().'.jpg';
```

Comment:
Ok at least I would move this logic in the factory two avoid the duplication between controllers by using a getter/setter for the picture path.